raster/src/main/scala/geotrellis/raster/EuclideanDistanceTile.scala

```diff
@@ -0,0 +1,29 @@
+package geotrellis.raster
+
+import geotrellis.raster.rasterize.polygon._
+import geotrellis.vector._
+import geotrellis.vector.voronoi._
+import scala.math.sqrt
+
+object EuclideanDistanceTile {
```

Comment:
This should be moved out of root package; `geotrellis.raster.distance` seems general enough to be a home for it.

Needs `MethodExtensions[Traversable[Point]]`, and the corresponding machinery.